src/main.js

```diff
@@ -66,14 +66,15 @@ export default class Auth {
 		// we need to check if this environment supports `addEventListener` on the window.
 		'addEventListener' in window &&
 			window.addEventListener('storage', e => {
-				// This code will run if localStorage for this user
-				// data was updated from a different browser window.
+				// This code will run if the localStorage for this user
+				// was updated from a different browser window.
 				if (e.key !== this.sKey('User')) return;
 				this.setState(JSON.parse(e.newValue), false);
 			});
 	}
 
 	/**
+	 * Emits an event and triggers all the listeners.
```

Comment:
duplicate line

Reply:
I removed the other one (line78), thanks.